modules/luponmediaBidAdapter.js

```diff
@@ -166,10 +315,26 @@ function newOrtbBidRequest(bidRequest, bidderRequest, currentImps) {
     }
   }
 
-  const bidFloor = parseFloat(utils.deepAccess(bidRequest, 'params.floor'));
+  let bidFloor;
+  if (typeof bidRequest.getFloor === 'function' && !config.getConfig('disableFloors')) {
+    let floorInfo;
+    try {
+      floorInfo = bidRequest.getFloor({
+        currency: 'USD',
+        mediaType: 'video',
+        size: parseSizes(bidRequest, 'video')
+      });
+    } catch (e) {
+      utils.logError('LuponMedia: getFloor threw an error: ', e);
+    }
+    bidFloor = typeof floorInfo === 'object' && floorInfo.currency === 'USD' && !isNaN(parseInt(floorInfo.floor)) ? parseFloat(floorInfo.floor) : undefined;
+  } else {
+    bidFloor = parseFloat(utils.deepAccess(bidRequest, 'params.floor'));
```

Comment:
We have made it so that all adapters now only import the needed "utils" functions so you no longer need `utils.` in the root of the function call anymore.